addons/insignias/config.cpp

```diff
@@ -8,7 +8,7 @@ class CfgPatches {
         requiredVersion = REQUIRED_VERSION;
         requiredAddons[] = {"tacs_main"};
         author = ECSTRING(main,Author);
-        authors[] = {"Jonpas", "BadHabitz", "System98"};
+        authors[] = {"Jonpas", "BadHabitz", "System98","Mick Mathews"};
```

Comment:
Space before his name, like before others.
Also did you ask him if he wants that name here or would rather prefer his nickname or something else?

Reply:
I asked him if he wanted his name or something else, he said his name